src/Components/CreateImageWizard/utilities/PasswordValidatedInput.tsx

```diff
@@ -61,14 +62,29 @@ export const PasswordValidatedInput = ({
             />
           </InputGroupItem>
           <InputGroupItem>
-            <Button
-              variant="control"
-              onClick={togglePasswordVisibility}
-              aria-label={isPasswordVisible ? 'Hide password' : 'Show password'}
-              isDisabled={hasPassword && !value}
-            >
-              {isPasswordVisible ? <EyeSlashIcon /> : <EyeIcon />}
-            </Button>
+            {hasPassword && !value ? (
+                <Tooltip content="Passwords cannot be viewed when editing a blueprint for security reasons">
+                <span>
+                  <Button
+                      variant="control"
+                      onClick={togglePasswordVisibility}
+                      aria-label={isPasswordVisible ? 'Hide password' : 'Show password'}
+                      isDisabled={true}
+                  >
+                    {isPasswordVisible ? <EyeSlashIcon /> : <EyeIcon />}
+                  </Button>
```

Comment:
Can you please move the button to a reusable component to remove the duplicate code?